Toolbox/ArcSDM.pyt

```diff
@@ -1203,52 +1203,59 @@ class SiteReductionTool(object):
 
     def getParameterInfo(self):
         """Define parameter definitions"""
-        param0 = arcpy.Parameter(
+        param_input_layer = arcpy.Parameter(
         displayName="Training sites layer",
         name="Training_Sites_layer",
         datatype="GPFeatureLayer",
         parameterType="Required",
         direction="Input")
 
-        param1 = arcpy.Parameter(
+        param_use_thinning_selection = arcpy.Parameter(
         displayName="Thinning selection",
         name="Thinning_Selection",
         datatype="Boolean",
         parameterType="Optional",
         direction="Input")
 
-        param2 = arcpy.Parameter(
+        param_unit_area = arcpy.Parameter(
         displayName="Unit area (sq km)",
         name="Unit_Area__sq_km_",
-        datatype="GPDouble",
+        datatype= ["GPDouble", "GPLong"],

```

Comment:
Now that we realized that this was likely just a localization issue, I think this can be returned to just "GPDouble".